contracts/gauges/gauge/src/state.rs

```diff
@@ -51,6 +49,10 @@ pub struct Gauge {
     pub adapter: Addr,
     /// Frequency (in seconds) the gauge executes messages, typically something like 7*86400
     pub epoch: u64,
+    /// Epoch count.
+    pub count: Option<u64>,
+    /// total possible count for a gauge to run. will automatially disable itself when reaching this epoch count.
```

Comment:
```suggestion
    /// Total possible count for a gauge to run. will automatially disable itself when reaching this epoch count.
```

I'm a millenial... we capitalize things. 😂

Reply:
Maybe also useful to describe behavior if this value isn't set.